### PR TITLE
CLDR-14389 make ToC in Dashboard clickable

### PR DESCRIPTION
**Pull request by srl295**
- scrolling isnt perfect, but approximates

### CODE REVIEW

File: tools/cldr-apps/js/src/views/DashboardPanel.vue [hunk 1]
```diff
@@ -106,6 +110,34 @@ export default {
...
+      const whence = event.target.getAttribute("notification");
+      if (this.data && this.data.notifications) {
+        for (let n of this.data.notifications) {
+          if (n.notification == whence) {
+            n.hidden = false;
+            const selector = `#notification-${whence}`;
+            try {
+              const scrollFix = event.target.offsetParent.offsetTop;
+              document.querySelector(selector).scrollIntoView(true);
+              // Now, scroll BACK because:
```
Comment (btangmu) [inline tools/cldr-apps/js/src/views/DashboardPanel.vue:123]: possible alternative: `scrollIntoView({ block: "center" })`

File: tools/cldr-apps/js/src/views/DashboardPanel.vue [hunk 2]
```diff
@@ -106,6 +110,34 @@ export default {
     this.fetchData();
   },
   methods: {
+    scrollto: function scrollto(event) {
+      const whence = event.target.getAttribute("notification");
```
Comment (btangmu) [inline tools/cldr-apps/js/src/views/DashboardPanel.vue:114]: I suspect `scrollto: function` is superfluous

**Review by btangmu — APPROVED** [review]

File: tools/cldr-apps/js/src/views/DashboardPanel.vue (hunk 2 above)
Comment (srl295) [reply tools/cldr-apps/js/src/views/DashboardPanel.vue:114]: Probably. Debugging an issue with "this".

File: tools/cldr-apps/js/src/views/DashboardPanel.vue (hunk 1 above)
Comment (srl295) [reply tools/cldr-apps/js/src/views/DashboardPanel.vue:123]: Like to get this in and iterate.

Comment (srl295) [reply tools/cldr-apps/js/src/views/DashboardPanel.vue:123]: Updating the browser hash would break as you might imagine.

**Pull request merged**